taxonomy-editor-frontend/src/pages/editentry/index.tsx

```diff
@@ -95,22 +101,23 @@ const EditEntry = ({ addNavLinks }) => {
           </Button>
         </DialogActions>
       </Dialog>
+
       {/* Dialog box for acknowledgement of deletion of node */}
       <Dialog
         open={openSuccessDialog}
         aria-labelledby="alert-dialog-title"
         aria-describedby="alert-dialog-description"
       >
         <DialogTitle id="alert-dialog-title">
-          {"Your edits have been saved!"}
+          Your edits have been saved!
         </DialogTitle>
         <DialogContent>
           <DialogContentText id="alert-dialog-description">
             The node {id} has been successfully deleted.
           </DialogContentText>
         </DialogContent>
         <DialogActions>
-          <Button component={Link} to={`${urlPrefix}/entry`} autoFocus>
+          <Button component={Link} to={`${taxonomyName}/${branchName}/entry`}>
```

Comment:
Please check if links go to their corresponding targets.
```suggestion
          <Button component={Link} to={`/${taxonomyName}/${branchName}/entry`}>
```